### DISCUSSION TITLE
We want to use ngrx/component, is it posible? since its in development still.

### DISCUSSION THREAD

**Discussion by sameh-g** [ngrx/platform#3619] [Q&A] [answered]
### Which @ngrx/* package(s) are relevant/related to the feature request?

component

### Information

We are aiming in the team to start using nrx/component, it says that it still under development. Is it possible to use it for production ?

### Describe any alternatives/workarounds you're currently using

We are aiming in the team to start using nrx/component, it says that it still under development. Is it possible to use it for production ?

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by brandonroberts** [comment, marked as answer]
Hey @sameh-g, yes you can start using it. It's been under development and is being used in production already. The current plan is to mark it as stable in the v15 release
